app/models/data_export.rb

```diff
@@ -84,6 +84,12 @@ class DataExport < ApplicationRecord
       description: 'Data related to notes made on applications by providers.',
       class: SupportInterface::NotesExport,
     },
+    notification_preferences_export: {
+      name: 'Notification preferences',
```

Comment:
Can we rename this as it clashes with the other export/pr?

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/4905/files#diff-2feb720f0c8b0d5b537b6599ecde84199e87558ef8536269ae7726d0c3b2d8dfR88